### PR TITLE
fix: cmfinder writing to inputdir

### PR DESCRIPTION
**Pull request by pavanvidem**
I didn't any datatype named `text`. Hence, changed from `format="text"` to `format="txt"`. Does it make sense?

### CODE REVIEW

**Review by bgruening — APPROVED** [review]
Yes, this makes sense.

**Comment by pavanvidem** [comment]
sorry, I forgot to update the tool version

File: tools/GraphClust/AlignCluster/align_cluster.xml [hunk 1]
```diff
@@ -102,12 +102,12 @@
...
               <param format="fasta" name="transcript_loci_fasta" type="data" label="Loci reference sequence"
                 help="Sequence of reference species (human) for the genomic loci (precursor mRNA/lncRNA) to locate the clusters" />
-              <param name='transcript_loci_bed' type="text" value="chr1 0 100000 gene 0 +" 
+              <param name='transcript_loci_bed' type="txt" value="chr1 0 100000 gene 0 +" 
                 help="bed entry string of the reference transcript loci for bed/ucsc-track output. 'chrom start end name score strand' white-space separated.">
                 <validator type="regex" message="one-line bed string must have at least 6 entries as BED6 format (chrom start end name 0 strand) 
                 (space and tab allowed)">^\S+\s+[0-9]+\s+[0-9]+\s+\S+\s+\S+\s+[-+]\s*$</validator> 
               </param>
-              <param name='genome_version' value="hg38" type="text" label="reference genome assembly"
+              <param name='genome_version' value="hg38" type="txt" label="reference genome assembly"
```
Comment (bgruening) [inline tools/GraphClust/AlignCluster/align_cluster.xml:110]: ah shit, @pavanvidem this was wrong. a parameter can hve the type="text"

A data-prameter can only have the format "txt"

File: tools/GraphClust/GSPAN/fasta2shrep_gspan.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
         <param name="M" type="integer" value="5" label="Max number of shreps that should be taken per window." help="-M"/>
         <param name="rel_energy_range" type="integer" value="20"
             label=" Relative energy range, i.e. percentage (%) of MFE energy (RNAshapes)" help="-c"/>
-        <param name="wins" type="text" value="40,150"
+        <param name="wins" type="txt" value="40,150"
```
Comment (bgruening) [inline tools/GraphClust/GSPAN/fasta2shrep_gspan.xml:38]: text

File: tools/GraphClust/Structure_GSPAN/structure_to_gspan.xml [hunk 3]
```diff
@@ -20,9 +20,9 @@
     </command>
     <inputs>
         <param type="data" name="dataFile" format="dbn" label="Sequence and Structure input file in the prediction tool format" />
-        <param name="inputFormat" type="text" value="vrna-simple"
+        <param name="inputFormat" type="txt" value="vrna-simple"
             label="  Sequence Structure format of the input. Allowed formats: vrna-simple, vrna-mea" help="-input-format"/>
-        <param name="structureType" type="text" value="MFE"
+        <param name="structureType" type="txt" value="MFE"
```
Comment (bgruening) [inline tools/GraphClust/Structure_GSPAN/structure_to_gspan.xml:25]: text

File: tools/GraphClust/Structure_GSPAN/structure_to_gspan.xml [hunk 4]
```diff
@@ -20,9 +20,9 @@
     </command>
     <inputs>
         <param type="data" name="dataFile" format="dbn" label="Sequence and Structure input file in the prediction tool format" />
-        <param name="inputFormat" type="text" value="vrna-simple"
+        <param name="inputFormat" type="txt" value="vrna-simple"
```
Comment (bgruening) [inline tools/GraphClust/Structure_GSPAN/structure_to_gspan.xml:23]: text

File: tools/GraphClust/AlignCluster/align_cluster.xml (hunk 1 above)
Comment (pavanvidem) [reply tools/GraphClust/AlignCluster/align_cluster.xml:110]: Sorry, my mistake. I accidentally replaced all `text` to `txt`. I will revert it

**Pull request merged**